packages/@jsii/java-runtime/project/src/main/java/software/amazon/jsii/JsiiRuntime.java

```diff
@@ -76,7 +76,7 @@ public final class JsiiRuntime {
      *
      * @param request The JSON request
      * @return The JSON response
-     * @throws JsiiException If the runtime returns an error response.
+     * @throws RuntimeException or JsiiError If the runtime returns an error response.
```

Comment:
This is not how you code up `@throws`:

```suggestion
     * @throws JsiiError If the runtime returns an error response originating from the @jsii/kernel.
     * @throws RuntimeException If the runtime returns an error response.
```